settings/src/script.js

```diff
@@ -61,6 +61,7 @@ function Main( { userId } ) {
 	} = user;
 	const [ globalNotice, setGlobalNotice ] = useState( '' );
 	const [ error, setError ] = useState( '' );
+	const [ backupCodesVerified, setBackupCodesVerified ] = useState( true );
```

Comment:
Just add some notes about my testing process for the logic here for future reference.

Initially, I thought of setting this value to `false` during testing. 
And setting [this part](https://github.com/WordPress/wporg-two-factor/pull/257/commits/f1e7bdee6bbb9e798f8a333e4ab063a872688d8c#diff-7eb8cf2231795f70529459a7212a92b4521001f290e37a74596a010b72e02914R33) to `true` only when the `Back` button on the backup-codes screen is pressed. 

However, there are two issues with this approach:

1. Currently, within `ScreenLink`, we can't determine which screen the `Back` button is pressed from.
2. Even if we could determine that, since the value is set to `true` only when the `Back` button on the backup-codes screen is pressed, [this condition](https://github.com/WordPress/wporg-two-factor/pull/257/commits/f1e7bdee6bbb9e798f8a333e4ab063a872688d8c#diff-8a318bcbbb9938100693a8ca195e7b0286195c925d93307290c776c74e073997R45) with `! backupCodesVerified` would be `true` again when the user exits the account screen (when `Main` is unmounted) and returns to the account screen, leading to another unexpected re-generation of backup-codes.

So, the current logic is that every time the `ScreenLink` component is pressed, it will execute `setBackupCodesVerified( true );`. I believe this is an acceptable compromise, especially since this workaround was put in place primarily due to the side effects of the side-effects of `two-factor/#507`.